.github/workflows/publish-pypi.yml

```diff
@@ -16,15 +16,15 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - uses: conda-incubator/setup-miniconda@v2
+      - uses: mamba-org/setup-micromamba@v1
         with:
-          activate-environment: csep-dev
-          python-version: '3.10'
-          channels: conda-forge
+          generate-run-shell: true
+          environment-file: requirements.yml
+          create-args: >-
+            python=3.10
 
```

Comment:
Looks good Pablo, this change from conda to mamba speeds-up processing quite a bit. I compared  this script against the other CI scripts and it looks good. This should run much faster with micromamba.